stoobly_agent/app/cli/helpers/endpoints_import_service.py

```diff
@@ -124,14 +145,19 @@ def import_response_header_name(project_id: int, endpoint_id: int, response_head
     ENDPOINT_COMPONENT_NAMES[4]: import_response_param_name
 }
 
-def process_import(component_name: str, *args):
-    return component_name_import_dispatch[component_name](*args)
+def process_import(component_name: str, body_param_parents: Dict[int, int], response_param_parents: Dict[int, int], *args):
+    if component_name == ENDPOINT_COMPONENT_NAMES[1]:
+        return component_name_import_dispatch[component_name](*args, body_param_parents)
+    elif component_name == ENDPOINT_COMPONENT_NAMES[4]:
+        return component_name_import_dispatch[component_name](*args, response_param_parents)
+    else:
+        return component_name_import_dispatch[component_name](*args)
 
-def import_component_names(project_id: int, endpoint_id: int, endpoint: EndpointShowResponse, resources: Dict[str, EndpointsResource]):
+def import_component_names(project_id: int, endpoint_id: int, endpoint: EndpointShowResponse, resources: Dict[str, EndpointsResource], body_param_parents: Dict[int, int], response_param_parents: Dict[int, int]):
```

Comment:
One suggestion to alleviate large number of function parameters is to use a context object and pass that around.